keyboards/crkbd/keymaps/gotham/config.h

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
If you could, run `qmk cformat keyboards/crkbd/keymaps/gotham/config.h`.  This should reformat the file with the QMK Coding Conventions (using CLANG). 

You may need to use `bin/qmk`, if that errors out.

Aside from that, I don't see anything wrong with the file, just a bunch of formatting nit picks.

Reply:
I'm having trouble with running this on Windows, sorry about that. I'll address this soon.

Reply:
Done!